src/main/java/com/ipiecoles/java/java220/Cadre.java

```diff
@@ -0,0 +1,160 @@
+package com.ipiecoles.java.java220;
+
+import org.joda.time.LocalDate;
+
+import java.util.Objects;
+
+public class Cadre extends Employe {
+    private Double coefficient;
```

Comment:
Tu peux aussi affecter l'attribut par défaut ici
```suggestion
    private Double coefficient = 1d;
```